commons/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="white">#FFFFFF</color>
```

Comment:
Wouldn't it be convenient to use android color white resource?

Reply:
Yes, I didn't remember that Android has internally those colors.